tests/test_response.py

```diff
@@ -280,6 +280,16 @@ def test_send_file_compressed(self):
                          'application/octet-stream')
         self.assertEqual(res.headers['Content-Encoding'], 'gzip')
 
+    def test_send_file_with_correct_mime_type_and_content_encoding(self):
+        res = Response.send_file('tests/files/test.txt')
+        self.assertEqual(res.status_code, 200)
+        self.assertEqual(res.headers['Content-Type'], 'text/plain')
+
+        res = Response.send_file('tests/files/test.txt.gz')
```

Comment:
This test should use octet-stream as content file, because the `compressed` argument is `False` by default.

There should be another test that passes `compressed=True` and only then the we want the content type to be text and the content-encoding header to be present.